### DISCUSSION TITLE
How To Use?

### DISCUSSION THREAD

**Discussion by nakheel77** [syntaxticsugr/ViPER4AndroidFXRepack-Presets#3] [Q&A] [answered]
Hi bro, many thabks for your work !

But i dont know how to use this ? Can you give me guide to install this ?

I already Viper4android, but i dont know how to setup kernel, preset, ddc...

Thank you very much !

**Comment by syntaxticsugr** [comment, marked as answer]
**Step 1.** Clear ViPER app data from device settings.

**Step 2.** From the [latest release](https://github.com/syntaxticsugr/ViPER4Android-Presets/releases/latest) download any zip file from these three -

| Full | Lite | Recommended |
| --- | --- | --- |
| All `Preset`s<br>All `Kernel`s<br>All `DDC`s | Unique `Preset`s<br>Required `Kernel`s<br>Required `DDC`s | Unique `Preset`s<br>Unique `Kernel`s<br>Unique `DDC`s |

**Step 3.** Extract the zip file and you will see three folders - `Kernel`, `DDC` & `Preset`

**Step 4.** Move these three folders in `Android/data/com.pittvandewitt.viperfx/files/`

Now open ViPER app and enjoy : )

**Reply by gitdexgit** [reply]
> **Step 1.** Clear ViPER app data from device settings.
> 
> **Step 2.** From the [latest release](https://github.com/syntaxticsugr/ViPER4Android-Presets/releases/latest) download any zip file from these three -
> Full 	Lite 	Recommended
> All `Preset`s
> All `Kernel`s
> All `DDC`s 	Unique `Preset`s
> Required `Kernel`s
> Required `DDC`s 	Unique `Preset`s
> Unique `Kernel`s
> Unique `DDC`s
> 
> **Step 3.** Extract the zip file and you will see three folders - `Kernel`, `DDC` & `Preset`
> 
> **Step 4.** Move these three folders in `Android/data/com.pittvandewitt.viperfx/files/`
> 
> Now open ViPER app and enjoy : )

for me it's in Root/data/user/0/com.aam.ciper4android/files. This is the apk, I got the location from app manager (the one in F-droid). do I proceed ? with extracting the folders and `adb push <the_3_folders` to the locoation Root/data/user/0/com.aam.ciper4android/files? because I don't have the Android/data/com.pittvandewitt.viperfx/files/ location.

and also there isThe option of import kernel in viper4android. 


I hope that those kernels and presets have the convolver I'm trying to reverse the audio so that my bluetooth earbuds work

**Reply by i4yush** [reply]
Just Copy that folder to emulated/0/android/data/

**Reply by gitdexgit** [reply]
> Just Copy that folder to emulated/0/android/data/

Oh hi, I stopped using ViPER4Android a long time ago. I switches to jamesDSP, the root version. It works like a charm and the default stuff already reverses the audio channels. This was a long time ago.
